test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -0,0 +1,123 @@
+from enum import Enum
+import time
+import sys
+from drivers.SquishDriver import *
+from drivers.SquishDriverVerification import *
+
+
+class SigningPhrasePopUp(Enum):
+    OK_GOT_IT_BUTTON: str = "mainWindow_Ok_got_it_StatusBaseText"
+
+
+class MainWalletScreen(Enum):
+    ADD_ACCOUNT_BUTTON: str = "mainWallet_Add_Account"
+    ACCOUNT_NAME: str = "mainWallet_Account_Name"
+
+    
+class AddAccountPopup(Enum):
+    SCROLL_BAR: str = "mainWallet_Add_Account_Popup_Main"
+    PASSWORD_INPUT: str = "mainWallet_Add_Account_Popup_Password"
+    ACCOUNT_NAME_INPUT: str = "mainWallet_Add_Account_Popup_Account_Name"
+    ADVANCE_SECTION: str = "mainWallet_Add_Account_Popup_Advanced"
+    TYPE_SELECTOR: str = "mainWallet_Add_Account_Popup_Type_Selector"
+    TYPE_WATCH_ONLY: str = "mainWallet_Add_Account_Popup_Type_Watch_Only"
+    TYPE_SEED_PHRASE: str = "mainWallet_Add_Account_Popup_Type_Seed_Phrase"
+    TYPE_PRIVATE_KEY: str = "mainWallet_Add_Account_Popup_Type_Private_Key"
+    ADDRESS_INPUT: str = "mainWallet_Add_Account_Popup_Watch_Only_Address"
+    PRIVATE_KEY_INPUT: str = "mainWallet_Add_Account_Popup_Private_Key"
+    ADD_ACCOUNT_BUTTON: str = "mainWallet_Add_Account_Popup_Footer_Add_Account"
+    SEED_PHRASE_INPUT_0: str = "mainWindow_Add_Account_Popup_Seed_Phrase_0"
+    SEED_PHRASE_INPUT_1: str = "mainWindow_Add_Account_Popup_Seed_Phrase_1"
+    SEED_PHRASE_INPUT_2: str = "mainWindow_Add_Account_Popup_Seed_Phrase_2"
+    SEED_PHRASE_INPUT_3: str = "mainWindow_Add_Account_Popup_Seed_Phrase_3"
+    SEED_PHRASE_INPUT_4: str = "mainWindow_Add_Account_Popup_Seed_Phrase_4"
+    SEED_PHRASE_INPUT_5: str = "mainWindow_Add_Account_Popup_Seed_Phrase_5"
+    SEED_PHRASE_INPUT_6: str = "mainWindow_Add_Account_Popup_Seed_Phrase_6"
+    SEED_PHRASE_INPUT_7: str = "mainWindow_Add_Account_Popup_Seed_Phrase_7"
+    SEED_PHRASE_INPUT_8: str = "mainWindow_Add_Account_Popup_Seed_Phrase_8"
+    SEED_PHRASE_INPUT_9: str = "mainWindow_Add_Account_Popup_Seed_Phrase_9"
+    SEED_PHRASE_INPUT_10: str = "mainWindow_Add_Account_Popup_Seed_Phrase_10"
+    SEED_PHRASE_INPUT_11: str = "mainWindow_Add_Account_Popup_Seed_Phrase_11"
+
+
+class StatusWalletScreen:
+    
+    def acceptSigningPhrase(self):
```

Comment:
Can we agree on python namings conventions?? We have now a mix of _camelCase_ and / or _lower_case_with_underscores_

Reply:
Updated to snakecase as python convention is to use snakecase dunno why I used camelcase :facepalm:

Reply:
> Updated to snakecase as python convention is to use snakecase dunno why I used camelcase 🤦

Habit!! ;)